src/cuter_cerl.erl

```diff
@@ -11,13 +11,14 @@
 -export([node_types_branches/0, node_types_branches_nocomp/0, node_types_all/0,
          node_types_conditions/0, node_types_conditions_nocomp/0,
          node_types_paths/0, node_types_paths_nocomp/0]).
-%% Exported for debugging use.
--export([classify_attributes/1]).
 %% kfun API.
--export([kfun/2, kfun_code/1, kfun_is_exported/1]).
+-export([kfun/2, kfun_code/1, kfun_is_exported/1, kfun_update_code/2]).
```

Comment:
An export of `kfun_update_code/2` was introduced, but it is not clear where this function is used in this PR.

Reply:
The reasoning is to ultimately merge the analysis in the branch annotations_rebased, but since it is rather large we are trying to spread it in manageable smaller pull requests. This function will be used at one of the next pull requests.

Reply:
OK, then the export should be in that PR...